doc/tutorial/introduction/introduction_align.cpp

```diff
@@ -1,17 +1,13 @@
-#include <seqan3/io/sequence_file/output.hpp>
-
-//! [sequence_input_include]
-#include <seqan3/core/debug_stream.hpp>      // for debug_stream
-#include <seqan3/io/sequence_file/input.hpp> // for sequence_file_input
-//! [sequence_input_include]
-
-//! [alignment_include]
-#include <tuple>                                                          // for std::make_pair
-#include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp> // for alignment stream operator
-#include <seqan3/alignment/pairwise/align_pairwise.hpp>                   // for align_pairwise
-//! [alignment_include]
-
-using seqan3::operator""_dna4;
+//! [alignment]
+#include <filesystem>                                                       // for tmp_dir
```

Comment:
You cannot use the regular `<filesystem>` header because it is not available on gcc7.
You need to use `<seqan3/std/filesystem>` instead. You probably don't have this error because you are using gcc8 or 9

Reply:
That helped. Thanks for the hint!